block/config.ml

```diff
@@ -1,6 +1,23 @@
 open Mirage
 
-let main = foreign "Unikernel.Main" (console @-> block @-> job)
+type shellconfig = ShellConfig
+let shellconfig = Type ShellConfig
+
+let config_shell = impl @@ object
+    inherit base_configurable
+    method configure i =
+      let dir = Info.root i in
+      Functoria_app.Cmd.run "dd if=/dev/zero of=%s/disk.img count=100000" dir
+    method clean i =
+      let dir = Info.root i in
+      Functoria_app.Cmd.run "rm %s/disk.img" dir
+    method module_name = "Functoria_runtime"
```

Comment:
have a default in `base_configurable` sounds sensible to me... it will break code, though (clients need `method !module_name` now).. I prepared a PR for functoria https://github.com/mirage/functoria/pull/62 -- let me know what you think (this is independent from this PR: this improves the block example for current released stack ; certainly block should be updated once (a released) functoria provides a default module_name)